### PR TITLE
add springboot-wildfly first version

### PR DESCRIPTION
**Pull request by arska**

### CODE REVIEW

File: app/springboot-wildfly.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Spring Boot Application in Wildfly
+==================================
+
+This example shows how to deploy a Spring Boot application with Wildfly to APPUiO. See the source code at:
+
+* https://github.com/arska/springdemo
```
Comment (tobru) [inline app/springboot-wildfly.rst]: Please move to https://github.com/appuio/springdemo

File: app/springboot-wildfly.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+We use the Maven-based source-to-image (s2i) build process for wildfly applications.
```
Comment (tobru) [inline app/springboot-wildfly.rst]: wildfly -> Wildfly (not only here...)

File: app/springboot-wildfly.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  oc process -f https://raw.githubusercontent.com/arska/springdemo/master/springdemo-template.json --parameters
+
+The resulting applications have labels with the appname associated with them, so to delete them again you can: ::
+  oc delete all -l appname=springdemo
+  oc delete all -l appname=second
+  oc delete all -l appname=third
+
+Or to delete all instances of the template: ::
+  oc delete all -l template=springdemo
+
```
Comment (tobru) [inline app/springboot-wildfly.rst:51]: Would it make sense to add a link to https://docs.openshift.org/latest/dev_guide/templates.html for further information about templates?

**Review by tobru — CHANGES_REQUESTED** [review]

**Comment by arska** [comment]
Updated, please review @tobru

**Pull request merged**